packages/commercetools-sdk-middleware-auth/README.md

```diff
@@ -0,0 +1,39 @@
+# commercetools-sdk-middlware-auth
+
+Auth middlewares collection for usage with `@commercetools/sdk`
```

Comment:
I would prefer `@commercetools/sdk` since we are talking about a 'node package manager' package.